### PR TITLE
Haroun home logic

### PR DESCRIPTION
**Pull request by menaili**

### CODE REVIEW

File: lib/app/modules/home_page/views/components/discover_algeria.dart [hunk 1]
```diff
@@ -3,44 +3,50 @@ import 'package:flutter_screenutil/flutter_screenutil.dart';
...
+          child: Row(
+            children: [
+              Text(
+                title,
+                style: TextStyles.mediumLabelTextStyle(context)
+                    .copyWith(color: MainColors.textColor(context)),
+              ),
+              Spacer(),
+              Text(
+                'See More',
```
Comment (farhioussama1969) [inline lib/app/modules/home_page/views/components/discover_algeria.dart]: translation

File: lib/app/modules/home_page/views/components/favorites_distinations.dart [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+          child: Row(
+            children: [
+              Text(
+                title,
+                style: TextStyles.mediumLabelTextStyle(context)
+                    .copyWith(color: MainColors.textColor(context)),
+              ),
+              Spacer(),
+              Text(
+                'See More',
```
Comment (farhioussama1969) [inline lib/app/modules/home_page/views/components/favorites_distinations.dart]: same

File: lib/app/modules/home_page/views/components/favorites_distinations.dart [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+                            ),
+                            maxLines: 2,
+                            textAlign: TextAlign.left,
+                          ),
+                          SizedBox(
+                            height: 4.h,
+                          ),
+                          Text(
+                            (itemsList[index].departuresCount ?? 0).toString() +
+                                ' establishment',
```
Comment (farhioussama1969) [inline lib/app/modules/home_page/views/components/favorites_distinations.dart]: same

File: lib/app/modules/home_page/views/components/list_filtre_view_component.dart [hunk 4]
```diff
@@ -13,6 +13,7 @@ class ListFiltreViewComponent extends StatelessWidget {
     return SizedBox(
       height: 100,
       child: ListView.builder(
+        padding: const EdgeInsets.symmetric(horizontal: 20.0),
```
Comment (farhioussama1969) [inline lib/app/modules/home_page/views/components/list_filtre_view_component.dart:16]: .w

File: lib/app/modules/home_page/views/components/organized_trip_component.dart [hunk 5]
```diff
@@ -5,44 +5,63 @@ import 'package:rihlatic/app/core/components/images/network_image_component.dart
...
+          child: Row(
+            children: [
+              Text(
+                title,
+                style: TextStyles.mediumLabelTextStyle(context)
+                    .copyWith(color: MainColors.textColor(context)),
+              ),
+              Spacer(),
+              Text(
+                'See More',
```
Comment (farhioussama1969) [inline lib/app/modules/home_page/views/components/organized_trip_component.dart]: same

File: lib/app/modules/home_page/views/components/organized_trip_component.dart [hunk 6]
```diff
@@ -294,7 +320,13 @@ class OrganizedTripComponent extends StatelessWidget {
                                 ),
                                 SizedBox(width: 2.w),
                                 Text(
-                                  '5 Hotel Amarina Sun Resort',
+                                  '5 ' +
```
Comment (farhioussama1969) [inline lib/app/modules/home_page/views/components/organized_trip_component.dart:324]: why 5?

File: lib/app/modules/home_page/views/components/organized_trip_component.dart [hunk 7]
```diff
@@ -322,7 +354,12 @@ class OrganizedTripComponent extends StatelessWidget {
                                 ),
                                 SizedBox(width: 8.w),
                                 Text(
-                                  '169000 DZD',
+                                  itemsList[index]
+                                          .departures!
+                                          .map((e) => e.priceIni ?? '')
+                                          .toList()
+                                          .join(' to ') +
+                                      ' DZD',
```
Comment (farhioussama1969) [inline lib/app/modules/home_page/views/components/organized_trip_component.dart]: .currency tranlasion

File: lib/app/modules/home_page/views/components/popular_flights.dart
Comment (farhioussama1969) [inline lib/app/modules/home_page/views/components/popular_flights.dart:1]: any components file should finished with name_component.dart

File: lib/app/modules/home_page/views/components/popular_flights.dart [hunk 8]
```diff
@@ -0,0 +1,171 @@
...
+          padding: const EdgeInsets.symmetric(horizontal: 20.0),
+          child: Row(
+            children: [
+              Text(
+                title,
+                style: TextStyles.mediumLabelTextStyle(context)
+                    .copyWith(color: MainColors.textColor(context)),
+              ),
+              Spacer(),
+              Text(
```
Comment (farhioussama1969) [inline lib/app/modules/home_page/views/components/popular_flights.dart]: ....

File: lib/app/modules/home_page/views/components/popular_flights.dart [hunk 9]
```diff
@@ -0,0 +1,171 @@
...
+                          topLeft: Radius.circular(16.r),
+                          topRight: Radius.circular(16.r),
+                          bottomLeft: Radius.circular(20.r),
+                          bottomRight: Radius.circular(20.r)),
+                      child: SizedBox(
+                        width: 242.w,
+                        height: 190.h,
+                        child: NetworkImageComponent(
+                          imageLink:
+                              'https://en.wikipedia.org/wiki/London#/media/File:London_Skyline_(125508655).jpeg',
```
Comment (farhioussama1969) [inline lib/app/modules/home_page/views/components/popular_flights.dart]: ???

Comment (menaili) [reply lib/app/modules/home_page/views/components/popular_flights.dart]: There is no picture in 'api'

**Pull request merged**